### PR TITLE
Doc: Fix example PROTO

### PR DESCRIPTION
**Pull request by ShuffleWire**
**Description**
Change the {} to []

### CODE REVIEW

**Review by omichel — DISMISSED** [review]
Well spotted!
Thank you.

Note: it would be better to target the `released` branch for this kind of fix as it would be published online as soon as the PR is merged. Otherwise, it will be published only after we make a new release of Webots.

File: docs/reference/proto-design-guidelines.md [hunk 1]
```diff
@@ -144,13 +144,13 @@ Here is a simple example of a good PROTO declaration (the implementation is not
 # license url: http://www.apache.org/licenses/LICENSE-2.0
 # A color pencil
 
-ColorPencil {
+ColorPencil [
```
Comment (stefaniapedrazzi) [inline docs/reference/proto-design-guidelines.md:147]:
```suggestion
PROTO ColorPencil [
```

File: docs/reference/proto-design-guidelines.md [hunk 2]
```diff
@@ -144,13 +144,13 @@ Here is a simple example of a good PROTO declaration (the implementation is not
...
   SFFloat    translation          0 0 0
```
Comment (stefaniapedrazzi) [inline docs/reference/proto-design-guidelines.md:148]: To be valid the we should also add the `field` keyword in each line.
(This field type was also wrong)
```suggestion
  field SFVec3f    translation          0 0 0
```

File: docs/reference/proto-design-guidelines.md [hunk 3]
```diff
@@ -144,13 +144,13 @@ Here is a simple example of a good PROTO declaration (the implementation is not
...
   SFRotation rotation             0 0 1 0
```
Comment (stefaniapedrazzi) [inline docs/reference/proto-design-guidelines.md:149]:
```suggestion
  field SFRotation rotation             0 0 1 0
```

**Review by stefaniapedrazzi — COMMENTED** [review]
Thank you!

However it is still not a valid PROTO.
This example is really a mess!

**Comment by ShuffleWire** [comment]
> Note: it would be better to target the `released` branch for this kind of fix as it would be published online as soon as the PR is merged. Otherwise, it will be published only after we make a new release of Webots.

Done !

**Review by stefaniapedrazzi — APPROVED** [review]
Thank you!

**Pull request merged**